src/main/java/com/softserve/kickscooter/kickscootervehicle/management/model/Scooter.java

```diff
@@ -0,0 +1,40 @@
+package com.softserve.kickscooter.kickscootervehicle.management.model;
+
+import lombok.Data;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+public class Scooter {
+
+    @Id
+    @GeneratedValue
+    private UUID id;
+
+    private String modelName;
+
+    private String serialNumber;
+
+    @CreatedDate
+    private LocalDateTime registerDate;
+
+    private LocalDate expiredDate;
+
+    @Enumerated(EnumType.STRING)
+    private ScooterStatus status;
+
+    private Double actualLongitude;
+
+    private Double actualLatitude;
+
+    private Short battery;
```

Comment:
Can `battery`, `actualLatitude`, `actualLongitude` be null?

Reply:
No, in database notnull constraint

Reply:
any reason to use wrapper types then?

Reply:
ok, why do you need wrapper types then?

Reply:
I have to use primitives?